<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewListwidget</class>
 <widget class="QWidget" name="NewListwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>121</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#NewListWidget
{
   border-image:none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">
QWidget#widget
{
   border-image:none;
	border-image: url(:/images/images/listwidget.png);
 
}
QWidget#NewListwidget
{

}


QToolButton:hover
{
	/*background-color: rgba(255, 255, 255,0);*/
}

QToolButton
{
   border-radius:3px;
}




</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QToolButton" name="topoViewButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="alarmQueryButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="switchManageButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="portMirrorButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="configManageButton">
        <property name="text">
         <string>配置管理</string>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
